### DISCUSSION TITLE
Vitest errors are shown with the wrong column:line number

### DISCUSSION THREAD

**Discussion by kyrisu** [remix-run/remix#6453] [Q&A]
Hi 👋,

I'm starting it as a discussion because I couldn't find any issue about it, so maybe there's something wrong with my setup. 
If I missed an issue, at this point I'm just happy to be called dumb and pointed in the right direction.

Steps to replicate: 

1. Create an indie stack (accept `npm install` prompt etc)

```sh
npx create-remix@latest --template remix-run/indie-stack
```

2. Break a test, for example: 

```diff
diff --git a/app/utils.test.ts b/app/utils.test.ts
index 7ffd94a..b8c53f2 100644
--- a/app/utils.test.ts
+++ b/app/utils.test.ts
@@ -1,7 +1,7 @@
 import { validateEmail } from "./utils";
 
 test("validateEmail returns false for non-emails", () => {
-  expect(validateEmail(undefined)).toBe(false);
+  expect(validateEmail(undefined)).toBe(true);
   expect(validateEmail(null)).toBe(false);
   expect(validateEmail("")).toBe(false);
   expect(validateEmail("not-an-email")).toBe(false);
```

The resulting error points to the wrong place in the source(map):

<img width="892" alt="CleanShot 2023-05-22 at 23 56 46@2x" src="https://github.com/remix-run/remix/assets/212681/3b17e449-a7c2-4f98-ba44-1624ddbbc274">

There are a few issues in vitest repo, but they all seem to be resolved. Any help here would be greately appreciated.

**Comment by stephenprater** [comment]
I'm having this same issue - starting with an empty vitest config doesn't seem to affect it.
